mainlyetcetera/rotten_tomatillos
# Feature/movie data fetch

## Is this a fix or a feature?
- feature
## What is the change/fix?
- added successful api call for fetching movie data
- successfully adjusted state after api call
- display movie titles
## Where should the reviewer start?
- `src/components/App/App.js`
## How should this be tested?
- pull down branch (not to main)
- run `npm start` while on the branch
- observe api call and titles
## This PR is related to which issues:
#5
